data/input_2022/Architecture/Results/node-wot.csv

```diff
@@ -0,0 +1,69 @@
+"ID","Status","Comment"
+"arch-action-dataschema","null", "Do not understand the assertion"
+"arch-action-functions","impl",
+"arch-action-process","impl",
+"arch-affordances","impl",
+"arch-binding-templates-known-subprotocol","impl",
+"arch-binding-templates-media-type-optional-parameters","impl",
+"arch-binding-templates-media-type-parameters","impl",
+"arch-binding-templates-subprotocol-explicit-declaration","impl",
+"arch-discovery-metadata-after-authorization","not-impl",
+"arch-discovery-no-metadata-in-introduction","not-impl",
+"arch-discovery-no-metadata-in-introduction-urls","not-impl", "User decision, URL based on interaction name"
+"arch-discovery-should-use-standard","not-impl",
+"arch-event-dataschema","null", "Do not understand the assertion"
+"arch-event-trigger","impl",
+"arch-form-iris","impl",
+"arch-form-iris2","impl",
+"arch-hypermedia","impl",
+"arch-hypermedia-caching","not-impl",
+"arch-hypermedia-protocol-binding","impl",
+"arch-intermediary-td-extra-protocols","null",
+"arch-media-type","impl",
+"arch-media-type-extra","impl",
+"arch-networked","impl",
+"arch-op-expected-request","impl",
+"arch-op-form-fields-protocol","impl",
+"arch-op-request-method","impl",
+"arch-other-thing-representations","not-impl", "May be done on top of it"
+"arch-privacy-consideration-access-control-mandatory-person","not-impl", "Use-case dependent"
+"arch-privacy-consideration-dist-td-auth","not-impl",
+"arch-privacy-consideration-explicit-pii","not-impl",
+"arch-privacy-consideration-id-access-control-mandatory-immutable","not-impl",
+"arch-privacy-consideration-min-explicit-pii","not-impl",
+"arch-privacy-consideration-no-unness-info-td","not-impl",
+"arch-property-dataschema","null", "Do not understand the assertion"
```

Comment:
I have a guess: Maybe it is about the option that you simple avoid a dataSchema definition since it would not provide any benefit to understand what is going on the wire. E.g., jpeg or mov data would not need a dataSchema definition.